src/fsipy/simulations/cylinder.py

```diff
@@ -1,3 +1,223 @@
+    print("Obtaining mesh, domains and boundaries...")
+    mesh = Mesh()
+    hdf = HDF5File(mesh.mpi_comm(), "mesh/" + mesh_path + ".h5", "r")
+    hdf.read(mesh, "/mesh", False)
+    boundaries = MeshFunction("size_t", mesh, 2)
+    hdf.read(boundaries, "/boundaries")
+    domains = MeshFunction("size_t", mesh, 3)
+    hdf.read(domains, "/domains")
+
+    return mesh, domains, boundaries
+
+
+class VelInPara(UserExpression):
+    def __init__(self, t, t_ramp, v_max_final, n, dsi, mesh, **kwargs):
+        self.t = t
+        self.t_ramp = t_ramp
+        self.v_max_final = v_max_final
+        self.v = 0.0
+        self.n = n
+        self.dsi = dsi
+        self.d = mesh.geometry().dim()
+        self.x = SpatialCoordinate(mesh)
+        # Compute area of boundary tesselation by integrating 1.0 over all facets
+        self.A = assemble(Constant(1.0, name="one") * self.dsi)
+        # Compute barycenter by integrating x components over all facets
+        self.c = [assemble(self.x[i] * self.dsi) / self.A for i in range(self.d)]
+        # Compute radius by taking max radius of boundary points
+        self.r = np.sqrt(self.A / np.pi)
+        super().__init__(**kwargs)
+
+    def update(self, t):
+        self.t = t
+        # apply a sigmoid ramp to the pressure
+        if self.t < self.t_ramp:
+            ramp_factor = -0.5 * np.cos(np.pi * self.t / self.t_ramp) + 0.5
+        else:
+            ramp_factor = 1.0
+        self.v = ramp_factor * self.v_max_final
+
+        if MPI.rank(MPI.comm_world) == 0:
+            print("v (centerline, at inlet) = {} m/s".format(self.v))
+
+    def eval(self, value, x):
+        r2 = (
+            (x[0] - self.c[0]) ** 2 + (x[1] - self.c[1]) ** 2 + (x[2] - self.c[2]) ** 2
+        )  # radius**2
+        fact_r = 1 - (r2 / self.r**2)
+
+        value[0] = -self.n[0] * (self.v) * fact_r  # *self.t
+        value[1] = -self.n[1] * (self.v) * fact_r  # *self.t
+        value[2] = -self.n[2] * (self.v) * fact_r  # *self.t
+
+    def value_shape(self):
+        return (3,)
+
+
+class InnerP(UserExpression):
+    def __init__(self, t, t_ramp, P_final, **kwargs):
+        self.t = t
+        self.t_ramp = t_ramp
+        self.P_final = P_final
+        self.P = 0.0
+        super().__init__(**kwargs)
+
+    def update(self, t):
+        self.t = t
+        # apply a sigmoid ramp to the pressure
+        if self.t < self.t_ramp:
+            ramp_factor = -0.5 * np.cos(np.pi * self.t / self.t_ramp) + 0.5
+        else:
+            ramp_factor = 1.0
+        self.P = ramp_factor * self.P_final
+
+        if MPI.rank(MPI.comm_world) == 0:
+            print("P = {} Pa".format(self.P))
+
+    def eval(self, value, x):
+        value[0] = self.P
+
+    def value_shape(self):
+        return ()
+
+
+def create_bcs(
+    dvp_,
+    DVP,
+    mesh,
+    boundaries,
+    P_final,
+    v_max_final,
+    fsi_id,
+    inlet_id,
+    inlet_outlet_s_id,
+    rigid_id,
+    psi,
+    F_solid_linear,
+    **namespace
+):
```

Comment:
I’m guessing you followed `flake8`, but changing line for each variables is a bit overkill and also makes overall script longer than necessary. You can have a few arguments and move to the next line when it becomes too long. For example, 
https://github.com/KVSlab/Aneurysm_Workflow_FSI/blob/268825f1639faea20bb4bf74c9a64965a42d01aa/src/fsipy/simulations/offset_stenosis.py#L158-L161